### ISSUE TITLE
Random Coloumns

### ISSUE THREAD

**Issue by nimit2801** [cli-table/cli-table3#211]
Description:
Random Columns are generated on the 2 last lines.
Is this error from my code or something internally in the lib?

**Code:** 
```js
let table = new Table(({
    head: ['Time', 'Mon', 'Tues', 'Wed', 'Thur', 'Fri']
}))

let period = ['Library', '.Net SK B1\nCG Lab VG B2\nDBMS SV B3', '.Net GC', 'TOC VP', 'DBMS SV', 'CG Lab VG B1\nDBMS SV B2\n.Net SK B3', 'IMP BP', 'CG YC','ID', 'DBMS SV B1\n.Net SK B2', 'CG LAB B3 VG', 'CG (TUT) YC', 'Club Activity', 'Mentor Meet'];

table.push(
    ['09:00 to 10:00', period[0], period[0], period[0], {rowSpan:3,content:period[1],vAlign:'center'}, period[2]],
    ['10:00 to 11:00', period[3], period[3], period[4], {rowSpan:3,content:period[5],vAlign:'center'}],
    ['11:00 to 12:00', period[6], period[7], period[3]],
    ['12:00 to 01:00', period[0], period[8], period[6], period[6]],
    ['01:00 to 02:00', {colSpan: 5, content: 'Lunch', hAlign: 'center'}],
    ['02:00 to 03:00', {rowSpan:3,content:period[9],vAlign:'center'},{rowSpan:3,content:period[10],vAlign:'center'}, period[3], period[2], period[4]],
    ['03:00 to 04:00', period[10], period[7], period[6]],
    ['04:00 to 05:00', period[3], period[11], period[12]]
);

console.log(table.toString())
```

**Output:** 
![image](https://user-images.githubusercontent.com/53462249/113536459-cfc02600-95f3-11eb-9b32-eeca185039ea.png)

**Comment by speedytwenty** [comment]
I think this is a duplicate of #128.

There's a PR that may resolve this for you: #187 

Please try the fix and upvote the PR if it works for you!

**Comment by speedytwenty** [comment]
I went ahead and tried nimit2801's code with the fix for #128 and it seems to resolve this issue:

![Screen Shot 2021-05-03 at 7 46 52 PM](https://user-images.githubusercontent.com/260874/116952693-7e35a600-ac48-11eb-9a94-432e8c4e3946.png)

**Comment by nimit2801** [comment]
oh, that's amazing @speedytwenty,
Thanks in then #187 solves #221.
